crates/bevy_log/src/lib.rs

```diff
@@ -12,7 +12,12 @@ pub use bevy_utils::tracing::{
 };
 
 use bevy_app::{AppBuilder, Plugin};
-use tracing_subscriber::{prelude::*, registry::Registry, EnvFilter};
+use tracing_subscriber::{
+    fmt::{format::DefaultFields, FormattedFields},
```

Comment:
These are marked unused by `cargo check`, causing CI to fail ):

Reply:
Fixed. Thanks.